src/test/java/seedu/address/model/item/field/TimeTest.java

```diff
@@ -0,0 +1,47 @@
+package seedu.address.model.item.field;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+public class TimeTest {
+    @Test
+    public void constructor_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new Time(null));
+    }
+
+    @Test
+    public void constructor_invalidTime_throwsIllegalArgumentException() {
+        String invalidTime = "";
+        assertThrows(IllegalArgumentException.class, () -> new Time(invalidTime));
+    }
+
+    @Test
+    public void isValidTime() {
+        // null time
+        assertThrows(NullPointerException.class, () -> Time.isValidTime(null));
```

Comment:
This should be in a different test method since it tests for exception and not proper return values.